executor/infoschema_reader.go

```diff
@@ -1836,18 +1836,27 @@ func (e *memtableRetriever) setNewTiKVRegionStatusCol(region *helper.RegionInfo,
 		} else {
 			row[6].SetInt64(0)
 		}
+		if table.IsPartition {
+			row[9].SetInt64(1)
+			row[10].SetInt64(table.Partition.ID)
+			row[11].SetString(table.Partition.Name.O, mysql.DefaultCollationName)
+		} else {
+			row[9].SetInt64(0)
+		}
+
```

Comment:
I think this is the reason for the error:
executor/infoschema_reader.go:1828:18: empty-lines: extra empty line at the end of a block (all_revive)
```suggestion
```